### ISSUE TITLE
Showing messed up tabs and unreadable sidebar

### ISSUE THREAD

**Issue by michaelaye** [jrolfs/sodarized#9]
What do you need from me to help you debug this? I'm looking forward to a consistent Solarized experience. ;)
FYI, I restarted ST, to no avail.
System: ST3 on OSX 10.10.5

<img width="1159" alt="screenshot 2015-08-17 18 56 03" src="https://cloud.githubusercontent.com/assets/69774/9319980/da411028-4511-11e5-9e35-470487e302c3.png">

**Comment by michaelaye** [comment]
Oh, and I have a iMac 5k Retina, so this might be an untested Retina support issue?

**Comment by cchamberlain** [comment]
@michaelaye I ran into something similar on Windows 10 at work, worked fine at home yesterday on Windows 10.  Looked exactly like yours all black (first screenshot was following package removal).  I tried changing the sublime-theme file to something non-existant, and it still looked all black and glyphy which led me to believe that the sublime-theme file might not have existed. I was able to resolve by using package control to install the correct sodarized package (one with jrolfs in the link) and that seems to fixed.  I think I may have accidentally installed the wrong package the first time.

I can try installing on my macbook retina when I get home tonight if you still got the issue.

![image](https://cloud.githubusercontent.com/assets/424694/10026028/51d280f4-6114-11e5-8665-3921ba9ecc46.png)

and after restarting ->

![image](https://cloud.githubusercontent.com/assets/424694/10026079/963473f6-6114-11e5-8fdd-e5f8f571a97e.png)
